internal/bin/bin.go

```diff
@@ -0,0 +1,104 @@
+// Copyright 2023 Google LLC
```

Comment:
This will be called from other deployers (i.e., GKE), so maybe it should go under `runtime/bin/`?

Reply:
Done.